code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm

```diff
@@ -50,10 +51,8 @@
 					infect_virus2(M, V.getcopy())
 
 /datum/reagent/blood/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(H.isSynthetic())
-			return
+	if(alien == IS_MACHINE)
```

Comment:
Unless defined in snacks.dm, IS_MACHINE isn't actually anything and in addition, it's something we don't use, as we don't have IPCs.